ru.toxblh.KeeneticManager.json

```diff
@@ -0,0 +1,60 @@
+{
+    "id" : "ru.toxblh.KeeneticManager",
```

Comment:
can you link your github profile somewhere on https://toxblh.com/?

Reply:
Yeah that linked here: https://toxblh.com/contacts/

Reply:
but the id is `ru.toxblh`.... not `com.*`

Reply:
https://toxblh.ru/contacts/ both is mine

Reply:
But I would like to use ru. version.